crates/aptos-inspection-service/src/server/memory_svg.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright © Aptos Foundation
+
+use crate::server::utils::CONTENT_TYPE_SVG;
+use hyper::{Body, StatusCode};
+use std::fs;
+
+pub fn handle_memory_svg_request() -> (StatusCode, Body, String) {
+    let content =
+        fs::read_to_string("./crates/aptos-inspection-service/src/server/memory_profile/heap.svg")
```

Comment:
how does that file get there, and why does it go there and not `.` or `/tmp` ?

Reply:
You're right, the file locations are to-be-changed as well. Will either store them in `/tmp` or another folder depending on if we want to store previous results long-term.